src/helpers/hideEmail/hideEmail.js

```diff
@@ -0,0 +1,14 @@
+const signToShow = '*';
+
+const replacer = (string, pattern1, pattern2) => (
+  pattern1.concat(signToShow.repeat(pattern2.length))
+);
+
+const hideEmail = email => (
+  email
+    .replace(/(^.{2})(.+?)(?=@)/g, replacer)
```

Comment:
Please add comments what is happening here.
And why do you do it in several steps?

Reply:
It is possible to replace in a single `String.prototype.replace` step but in that case we should implement a  more complicated `replacer` function with more pattern groups, it's signature is likely to be as `function replacer (string, p1, p2, p3, p4, p5, p6, p7)`  which is, from my point of view, less understandable. What do you think about it?